.travis.yml

```
@@ -73,9 +73,17 @@ jobs:
       os: windows
       language: shell
       before_install:
-        - docker build -t cocotb -f Dockerfile.windows .
+        - powershell "Invoke-WebRequest -outfile miniconda3.exe https://repo.continuum.io/miniconda/Miniconda3-latest-Windows-x86_64.exe"
+        - powershell "Start-Process .\miniconda3.exe -ArgumentList '/S /D=C:\miniconda3' -Wait"
+
+        - powershell "Invoke-WebRequest -Uri http://bleyer.org/icarus/iverilog-v11-20190809-x64_setup.exe -OutFile iverilog-x64_setup.exe"
+        - powershell "Start-Process .\iverilog-x64_setup -ArgumentList '/VERYSILENT' -Wait"
+
+        - export CONDA_PATH="\$env:Path = ';C:\miniconda3;C:\miniconda3\Library\mingw-w64\bin;C:\miniconda3\Library\usr\bin;C:\miniconda3\Library\bin;C:\miniconda3\Scripts;C:\miniconda3\bin;C:\miniconda3\condabin;C:\iverilog\bin;' + \$env:Path"
```

Can you set this env var directly, so that you can avoid some of this quoting?
```suggestion
        - export PATH="/C/miniconda3:/C/miniconda3/Library/mingw-w64/bin:/C/miniconda3/Library/usr/bin:/C/miniconda3/Library/bin:/C/miniconda3/Scripts:/C/miniconda3/bin:/C/miniconda3/condabin:/C/iverilog/bin:$PATH'
```

---

Actually, for whatever reason, this does not work (i tried ~10 versions) see: https://travis-ci.org/themperek/cocotb/jobs/645871415